NewsScraper/DawnCrawler/DawnSpiders/spiders/dawn.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from scrapy import Item
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+""" 
+   the item schema
+"""
+
+
+class DawnscraperItem(Item):
+    title = scrapy.Field()
+    image_urls = scrapy.Field()
+    published_date = scrapy.Field()
+    updated_date = scrapy.Field()
+    authors = scrapy.Field()
+    tweets = scrapy.Field()
+    comments = scrapy.Field()
+    content = scrapy.Field()
+    category = scrapy.Field()
+
+
+class DawnSpider(CrawlSpider):
+    name = 'dawn'
+    allowed_domains = ['dawn.com']
+    start_urls = [
+        "https://www.dawn.com/latest-news",
+        "https://www.dawn.com/business",
+        "https://www.dawn.com/opinion",
+        "https://www.dawn.com/prism",
+        "https://www.dawn.com/sport",
+        "https://www.dawn.com/magazines",
+        "https://www.dawn.com/world",
+        "https://www.dawn.com/tech",
+        "https://www.dawn.com/multimedia",
+        "https://www.dawn.com/in-depth/"
+    ]
+
+    rules = (
+        Rule(
+            LinkExtractor(allow='.*/news/.*'),
+            callback='parse_items', follow=True),
+
+    )
+
+    def parse_items(self, response):
+
```

Comment:
Extra line space

Reply:
Please make sure you fix all previous comments.